### DISCUSSION TITLE
Is this normal meshing behaviour?

### DISCUSSION THREAD

**Discussion by TimBeekman** [Koenkk/zigbee2mqtt#19368] [General]
Previously I used a Conbee II. Unfortunately there were some problems with devices not responding, what seems to be caused by the Conbee firmware. I changed over to the Sonoff zigbee dongle.

Below I've got 2 pictures. The one with the blue background is a topology map of the network with the Conbee stick. The other one is with the Sonoff dongle. It looks like meshing doesn't work as before. Is this normal?

![image](https://github.com/Koenkk/zigbee2mqtt/assets/89469564/2246dcbe-2232-43ef-95ec-5c4ac6651589)

![image](https://github.com/Koenkk/zigbee2mqtt/assets/89469564/530d1902-d6e2-482d-8a58-dd7879b4c796)

**Comment by francisp2** [comment]
How long has the mesh been up ?

**Reply by TimBeekman** [reply]
> How long has the mesh been up ?

24 hours

**Comment by TimBeekman** [comment]
Friday I've paired all of the devices to the Sonoff dongle. Today (monday) the meshing still looks odd. Routers only pair with 1 other router. Devices are slow to respond, and time-out messages in the logs.

What do I need to do to recover this network, as i don't believe time is fixing the issue.

**Comment by TimBeekman** [comment]
Short update;

When i mark "None of the above" i do see there are connections between to routers. But these are not sibling connections. Am i mistaken to think that routers meshing with other routers are siblings?
